src/main/java/com/minecolonies/coremod/event/FMLEventHandler.java

```diff
@@ -52,10 +52,6 @@ public static void onPlayerLogin(@NotNull final PlayerEvent.PlayerLoggedInEvent
             IColonyManager.getInstance().getIColonyByOwner(((ServerPlayerEntity) event.getPlayer()).getServerWorld(), event.getPlayer());
             //ColonyManager.syncAllColoniesAchievements();
         }
-        else
-        {
-            IColonyManager.getInstance().resetColonyViews();
-        }
```

Comment:
I added this for when switching worlds in singlePlayer, don't we need this?

Reply:
moved it to world unload instead, to not interfere with us sending views on login